internal/tsc_wrapped/path.ts

```diff
@@ -0,0 +1,21 @@
+import * as path from 'path';
+
+// resolve function that maintains backslashes
```

Comment:
'maintains' seems like the wrong word for these? do you mean "maintains forward slashes"?

I'm curious if we can find code inside the TypeScript compiler that does approx. the same thing, which would validate this.

They have compilerHost.canonicalFilename for this purpose, maybe we should actually be calling that instead of manipulating the slashes?

Reply:
Yup. I meant forward slashes. Updated to `preserves forward slashes on Windows`